docs/Configuration/agent-configuration.md

```diff
@@ -212,6 +234,42 @@ If you are using a self-hosted TUF server, you must also manage all of Fleetd's
 
 Fleet recommends deploying extensions created with osquery-go or natively with C++, instead of Python. Extensions written in Python require the user to compile it into a single packaged binary along with all the dependencies.
 
+### Targeting extensions with labels
+
+_Available in Fleet Premium v4.38.0_
+
+Fleet allows to target extensions to hosts that belong to specific labels. To set the labels you need to define a `labels` list under the extension name. 
+The label names in the list:
+- Must exist (otherwise the `/api/latest/fleet/config` request will fail).
+- Are case insensitive.
+- Extensions are deployed to hosts that are member of **all** the defined labels.
```

Comment:
A few suggested changes to this part for additional clarity + consistent formatting:

```suggestion
Fleet allows you to target extensions to hosts that belong to specific labels. To set these labels, you'll need to define a `labels` list under the extension name.
The label names in the list:
- must already exist (otherwise the `/api/latest/fleet/config` request will fail).
- are case insensitive.
- must **all** apply to a host in order to deploy the extension to that host.
```